tracim/tracim/lib/webdav/sql_resources.py

```diff
@@ -215,14 +215,24 @@ def createEmptyResource(self, file_name: str):
         if resource:
             content = resource.content
 
-        return FakeFileStream(
-            file_name=file_name,
-            content_api=self.content_api,
-            workspace=self.workspace,
-            content=content,
-            parent=self.content,
-            path=self.path + '/' + file_name
-        )
+        if not content:
+            # INFO - G.M - 21-03-2018 create new empty file and commit it.
+            new_empty_file = FakeFileStream(
+                file_name=file_name,
+                content_api=self.content_api,
+                workspace=self.workspace,
+                content=content,
+                parent=self.content,
+                path=path,
+            )
+            new_empty_file.close()
```

Comment:
Is it necessary to use FakeFileStream ? If the goal is to create a Content, maybe just use ContentLib ?

Reply:
I'm don't think so, this MR is actually more a proof of concept, i will refactor this. i'm thinking about removing all or most FakeFileStream class  and add his code directly into File (dav_resource).

Reply:
Ok. So open an issue for that.